meson.build

```diff
@@ -3669,6 +3669,10 @@ if conf.get('ENABLE_REPART') == 1
                 test('test-repart',
                      test_repart_sh,
                      args : [exe.full_path(), udevadm.full_path()])
+
+                if install_tests
+                        install_data(test_repart_sh.full_path(), install_dir : testsdir)
```

Comment:
please use path() instead of full_path(), it seems to be supported since meson 0.55.0. See #23391.